src/main/java/com/moddy/server/controller/model/dto/request/ModelCreateRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.moddy.server.controller.model.dto.request;
+
+import java.util.List;
+
+public record ModelCreateRequest(
+        String name,
+        String year,
+        String gender,
```

Comment:
```suggestion
        Gender gender,
```
p1
Gender enum class 내부에 정의된 메서드를 통해서 
String 값을 enum으로 변환하는 로직이 없어도
Gender 값을 request dto에 넣을 수 있습니다!